scripts/pre_commit_linter.py

```diff
@@ -1540,30 +1540,30 @@ def _check_sorted_dependencies(all_files, parsed_js_files):
                             'Please ensure that %s in %s, the injected '
                             'dependecies should be in the following manner:'
                             ' dollar imports, regular imports and constant '
-                            'imports, all in sorted form.'
+                            'imports, all in sorted order.'
                             % (property_value, filename))
                     if sorted_imports != literal_args:
                         failed = True
                         print (
-       	                    'Please ensure that %s in %s, the dependecies '
-       	                    'literally mentioned should be in the following '
-       	                    ' manner: dollar imports, regular imports and '
-       	                    'constant imports, all in sorted form.'
+       	                    'Please ensure that %s in %s, the stringfied '
+       	                    'dependencies should be in the following '
+       	                    'manner: dollar imports, regular imports and '
+       	                    'constant imports, all in sorted order.'
        	                    % (property_value, filename))
 
-    if failed:
-        summary_message = (
-            '%s   Sorted dependencies check failed' % (
-                _MESSAGE_TYPE_FAILED))
-        print summary_message
-        summary_messages.append(summary_message)
-    else:
-        summary_message = (
-            '%s  Sorted dependencies check passed' % (
-                _MESSAGE_TYPE_SUCCESS))
-        print summary_message
-        summary_messages.append(summary_message)
+    with _redirect_stdout(_TARGET_STDOUT):
+        if failed:
+            summary_message = (
+                '%s  Sorted dependencies check failed' % (
+                    _MESSAGE_TYPE_FAILED))
+            summary_messages.append(summary_message)
```

Comment:
Read the previous comment _carefully_: pull out this line as well.

Reply:
Done